### DISCUSSION TITLE
Community Website: Add Server Status Block Widget

### DISCUSSION THREAD

**Discussion by MisterAlphaBob** [gameserverapp/Platform#901] [Suggestions]
With the Ark servers having the inability to utilize the "Click to Join" button in the Server Block widget, the widget's only real use is to provide live updates on number of players in an Ark server. 

Suggestion: Provide a widget similar to the Discord Status Card, but for the Community Website. It could provide relevant real time data on all servers in a cluster such as Server Name, number of online players, Status of Server (online, offline, restarting, updating), etc.

**Comment by mvaessen** [comment]
Hi, you mean info like this? 
![Screenshot 2024-08-16 at 16 57 25](https://github.com/user-attachments/assets/7811b360-11c7-4feb-9074-fe37f5c18525)

**Reply by MisterAlphaBob** [reply]
Yes, but perhaps without all the space taken up with things like the button (which doesn't work for ark servers) and the text below. Instead replace with information more relevent to the player. I created a mockup up of what I am thinking since I am typically horrible at explaining my ideas hahahaha

![Proposed Widget](https://github.com/user-attachments/assets/4ca81ced-3f44-4ed0-b938-d3e9fd2c6563)

**Reply by MisterAlphaBob** [reply]
I know it's not the biggest deal, but I am just thinking of alternatives to providing a button to players that confuses them when they click it because it won't connect them. Players for Ark may not know that the button doesn't work because of the game. I understand this probably doesn't apply to other GSA games and is niche to Ark.

**Comment by goosenation** [comment]
Aside from the (nicely styled) info on that widget you designed, you can at least hide the button from the viewport using CSS. This bit will do the trick, add it to the website theme options:
```css
body.v3 .server-block .server-block-content .join-btn {
    position: relative;
    left: -100vw;
    opacity: 0;
}
```
This will still retain the curren block's height. If you would like to make the block less high, remove the space it still occupies with this snippet instead:
```css
body.v3 .server-block .server-block-content .join-btn {
    display: none;
}
```

**Reply by MisterAlphaBob** [reply]
Thank you! 🙂
